roles/oneagent/README.md

```diff
@@ -41,7 +41,7 @@ The following variables are available in `defaults/main/` and can be overridden:
 
 | Name | Default | Description
 |-|-|-
-| `oneagent_environment_url` | `-` | The URL of the target Dynatrace environment (SaaS or Managed).
+| `oneagent_environment_url` | `-` | The URL of the target Dynatrace environment (SaaS or Managed), see [Direct download from your environment](#direct-download-from-your-environment) for format.
```

Comment:
1. The URL of the target Dynatrace environment (see [Direct download from your environment](#direct-download-from-your-environment))